README.md

```diff
@@ -82,6 +82,13 @@ Applying mesos slave constraints (Optional)
 "id" may be type of range. 
 ```
 
+System Environment for Configurations
+--------------------------
+Many scheduler configurations can be made by setting the system environment variables.  To do this upper case the property and replace `.` with `_`.  
```

Comment:
I'd say "convert the property to upper case" instead of "upper case the property"